CHANGELOG.md

```diff
@@ -26,5 +26,6 @@
 * [ENHANCEMENT] flagext: for cases such as `DeprecatedFlag()` that need a logger, add RegisterFlagsWithLogger. #80
 * [ENHANCEMENT] Added option to BasicLifecycler to keep instance in the ring when stopping. #97
 * [ENHANCEMENT] Add WaitRingTokensStability function to ring, to be able to wait on ring stability excluding allowed state transitions. #95
+* [ENHANCEMENT] Trigger metrics update instead of doing it periodically to speed up tests that wait for certain metrics. #107
```

Comment:
[nit]
```suggestion
* [ENHANCEMENT] Trigger metrics update on ring changes instead of doing it periodically to speed up tests that wait for certain metrics. #107
```

Reply:
ack